### ISSUE TITLE
Installing in a new react-native project: missing yarn.lock

### ISSUE THREAD

**Issue by loredanacirstea** [berty/weshnet-expo#23]
tldr;
Add the `yarn.lock` file to the npm published package.
Otherwise, if you use this package in a new RN app, you will need to manually add it. But if you `cd ./node_modules/@berty/weshnet-expo && yarn` you may encounter some bundling issues, so I just copied `yarn.lock` from the `@berty/weshnet-expo` repo and it worked.

- new expo project, I installed `npx expo install @berty/weshnet-expo` and then rebuilt ios/android:

```bash
$ npx expo prebuild --clean
✔ Cleared android, ios code
✔ Created native directories
✔ Updated package.json | no changes
✔ Finished prebuild
⚠️  Something went wrong running `pod install` in the `ios` directory.
Command `pod install` failed.
└─ Cause: /bin/bash -c
set -e
make -C .. build.ios

make: *** No rule to make target `yarn.lock', needed by `node_modules'.  Stop.

pod install --repo-update --ansi exited with non-zero code: 1
```

I see that `node_modules/@berty/weshnet-expo` does not contain `yarn.lock`, which seems to be required by the build process in `node_modules/@berty/weshnet-expo/Makefile`.

So I did a `yarn` in `node_modules/@berty/weshnet-expo`, I got the following errors:

```bash
$ expo-module prepare && sh prepare.sh
Configuring module
expo-module-scripts: README.md exists, not updating
src/api/index.ts:1:10 - error TS2305: Module '"./protocoltypes.pb"' has no exported member 'default'.

1 import { default as protocolpb } from './protocoltypes.pb'
           ~~~~~~~

src/api/index.ts:2:10 - error TS2305: Module '"./rpcmanager.pb"' has no exported member 'default'.

2 import { default as rpcmanagerpb } from './rpcmanager.pb'
           ~~~~~~~

src/error.ts:2:8 - error TS1192: Module '"/Users/user/mobile/amigo/amigo/node_modules/@berty/weshnet-expo/src/api/index"' has no default export.

2 import types from './api/index.d'
         ~~~~~

src/rpc/rpc.bridge.ts:6:8 - error TS1192: Module '"/Users/user/mobile/amigo/amigo/node_modules/@berty/weshnet-expo/src/api/index"' has no default export.

6 import types from '../api/index.d'
         ~~~~~

src/weshnet.types.gen.ts:2:8 - error TS1192: Module '"/Users/user/mobile/amigo/amigo/node_modules/@berty/weshnet-expo/src/api/index"' has no default export.

2 import api from './api/index.d'
         ~~~
Found 5 errors in 4 files.

Errors  Files
     2  src/api/index.ts:1
     1  src/error.ts:2
     1  src/rpc/rpc.bridge.ts:6
     1  src/weshnet.types.gen.ts:2
error Command failed with exit code 2.
```

If I ignore the above errors, `npx expo prebuild --clean` executes without errors. But running on the ios simulator gives a runtime error, because the build process did not finalize and the `pb.js` files were not included in the build.

```
iOS Bundling failed 4672ms node_modules/expo/AppEntry.js (752 modules)
Unable to resolve "./protocoltypes.pb" from "node_modules/@berty/weshnet-expo/build/api/index.js"
```

So I reinstalled `weshnet-expo`, copied the `yarn.lock` from this repo, rebuilt and the ios app seems to work correctly.

**Comment by D4ryl00** [comment]
Hello, thanks for your issue.
We have an entry in .npmignore to don't ignore yarn.lock. I'll dig why it is excluded.

**Comment by jefft0** [comment]
Hello @loredanacirstea . Please see the latest commit which should resolve your issue.

**Comment by loredanacirstea** [comment]
with the latest version I get
```
iOS Bundling failed 7931ms node_modules/expo/AppEntry.js (1734 modules)
Unable to resolve "./protocoltypes.pb" from "node_modules/@berty/weshnet-expo/build/api/index.js"
```
at `npx expo prebuild --clean` the `build/api/*.pb.js` files are removed, even though they are present after `npm install`

I assume it is because a new build is forced, but there are bundling errors, that I mentioned in my original issue.

**Comment by bluesliverx** [comment]
I have the exact same problem as @loredanacirstea currently. Copying the pb files back into `node_modules/@berty/weshnet-expo/build/api` fixes the issue, but this should definitely be fixed.

**Comment by jefft0** [comment]
The following commands work for me. Can you give the exact commands which cause a problem?
```
npx create-expo-app@latest testexpo
cd testexpo
npx expo install @berty/weshnet-expo
npx expo prebuild --clean -p ios
```

**Comment by loredanacirstea** [comment]
@jefft0 I can confirm that those commands are successful. The issue is at runtime.

Add in `_layout.tsx`: 
```
import * as WeshnetExpo from "@berty/weshnet-expo";
console.log("WeshnetExpo",  WeshnetExpo.init())
```
Then `npm run ios` and you will see the error on the device/simulator.
